nestjs/src/courses/mentors/dto/mentor-student.dto.ts

```diff
@@ -11,8 +11,12 @@ export class MentorStudentDto extends StudentDto {
   constructor(student: Student) {
     super(student);
     this.feedbacks = student.feedbacks?.filter(f => f.mentorId === student.mentorId).map(f => ({ id: f.id })) ?? [];
+    this.repoUrl = student.repository ? student.repository : '';
```

Comment:
minor: it would be better to return `null` rather then empty string

Reply:
fixed